test/geometry/liegroup/test_so2.py

```diff
@@ -11,15 +11,17 @@ def _make_rand_data(self, device, dtype, input_shape):
         shape = input_shape[1:] if batch_size is None else input_shape
         return torch.rand(shape, device=device, dtype=dtype)
 
-    def test_smoke(self, device, dtype):
-        z = torch.randn(2, 1, dtype=torch.cfloat, device=device)
+    @pytest.mark.parametrize("cdtype", (torch.cfloat, torch.cdouble))
+    def test_smoke(self, device, dtype, cdtype):
```

Comment:
```suggestion
    def test_smoke(self, device, cdtype):
```
I think this will prevent pytest to generate unused cases